taskcall/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dd54da03-0a8c-4796-aaa6-61eeb04e611b",
+  "app_id": "taskcall",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and centralize Datadog incidents with TaskCall",
+    "title": "TaskCall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Incident Notifications",
+        "image_url": "images/incident_notifications.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Incident Details",
+        "image_url": "images/incident_details.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Impact Visibility and Status Dashboard",
+        "image_url": "images/impact_visibility_status_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "On-Call Management",
+        "image_url": "images/on_call_management.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Alerting",
+      "Category::Collaboration",
+      "Category::Incidents",
+      "Category::Issue Tracking",
+      "Category::Notification",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Submitted Data Type::Metrics",
+      "Queried Data Type::Events"
```

Comment:
Hi @riasat-ullah, I had suggested these as pure examples to show what the formatting of these classifier tags look like. Can you please confirm that the integration is sending metrics into Datadog and querying events out of Datadog? If not, can you please update accordingly? Thanks

Reply:
Oh! I did not realize that. We are not submitting any metrics or querying from Datadog. We only update the incidents that are sent to us by webhooks. As per @bgoldberg122's suggestion we removed the assets section, metadata.csv and service_checks.json files. I am assuming that with those out of the way, we can do without the Submitted Data Type and Queried Data Type. I will remove them.